_posts/2018-01-13-Day-1-Question-and-Solution.md

```diff
@@ -0,0 +1,52 @@
+---
+published: true
+---
+# Day 1
+# Issue 1
+
+## **Question:**
+## **Given three positive integers X, Y and P. Here P denotes the number of turns. Whenever the turn is odd X is multiplied by 2 and in every even turn Y is multiplied by 2. The task is to find the value of max(X, Y) ÷ min(X, Y) after the complete P turns.**
+
+## **Input:**
+## **The first line of the input contains a single integer T, denoting the number of test cases. Then T test case follows, a single line of the input containing three integers X,Y and P.**
+
+## **Constraints:**
+## **1<=T<=100**
+## **1<=X<=Y<=P<=1000**
+
+## **One of the solution:**
+	
+### package main
```

Comment:
`###` is for Heading as well any number of hashed from 1 to 6.

For code highlighting you have to use tilda charcter `

Read More about markdown from here - [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)

Reply:
Highlight code properly and Change `##` proper paragraphs